config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm.yaml

```diff
@@ -115,50 +62,52 @@ periodics:
     containers:
     - image: gcr.io/k8s-testimages/e2e-kubeadm:v20180829-29d5569b7
       args:
-      - --repo=k8s.io/kubernetes=release-1.8
+      - --repo=k8s.io/kubernetes=release-1.12
       - --repo=k8s.io/kubernetes-anywhere=kubeadm-e2e
       - --timeout=320
       - --upload=gs://kubernetes-jenkins/logs
       - --scenario=kubernetes_e2e
       - --
       - --cluster=
       - --deployment=kubernetes-anywhere
-      - --extract=ci/latest-bazel-1.8
+      - --extract=ci/latest-bazel-1.12
       - --gcp-zone=us-central1-f
       - --kubeadm=ci
-      - --kubernetes-anywhere-kubelet-ci-version=latest-bazel-1.8
-      - --kubernetes-anywhere-kubernetes-version=ci/latest-1.8
+      - --kubernetes-anywhere-kubelet-ci-version=latest-bazel-1.12
+      - --kubernetes-anywhere-kubernetes-version=ci/latest-1.12
       - --provider=kubernetes-anywhere
       - --test_args=--ginkgo.focus=\[Conformance\]|\[Feature:BootstrapTokens\]|\[Feature:NodeAuthorizer\] --minStartupPods=8
       - --timeout=300m
 
-- name: ci-kubernetes-e2e-kubeadm-gce-1-8-on-1-9
-  interval: 12h
+- name: ci-kubernetes-e2e-kubeadm-gce-master
+  interval: 2h
   labels:
     preset-service-account: "true"
     preset-k8s-ssh: "true"
   spec:
     containers:
     - image: gcr.io/k8s-testimages/e2e-kubeadm:v20180829-29d5569b7
       args:
-      - --repo=k8s.io/kubernetes=release-1.9
+      - --repo=k8s.io/kubernetes=master
       - --repo=k8s.io/kubernetes-anywhere=kubeadm-e2e
       - --timeout=320
       - --upload=gs://kubernetes-jenkins/logs
       - --scenario=kubernetes_e2e
       - --
+      # TODO(bentheelder): --kubernetes-anywhere-kubernetes-version should really be ci/latest-bazel but we need changes to kubeadm and possibly kubernetes-anywhere first
       - --cluster=
       - --deployment=kubernetes-anywhere
-      - --extract=ci/latest-bazel-1.8
+      - --extract=ci/latest-bazel
       - --gcp-zone=us-central1-f
       - --kubeadm=ci
-      - --kubernetes-anywhere-kubelet-ci-version=latest-bazel-1.8
-      - --kubernetes-anywhere-kubernetes-version=ci/latest-1.8
+      - --kubernetes-anywhere-kubelet-ci-version=latest-bazel
+      - --kubernetes-anywhere-kubernetes-version=ci-cross/latest
       - --provider=kubernetes-anywhere
       - --test_args=--ginkgo.focus=\[Conformance\]|\[Feature:BootstrapTokens\]|\[Feature:NodeAuthorizer\] --minStartupPods=8
       - --timeout=300m
 
-- name: ci-kubernetes-e2e-kubeadm-gce-1-9
+# kubeadm x-on-y tests
```

Comment:
perhaps we should split this into seperate config files, one for each of these commented blocks

Reply:
thanks, will update in a bit.